workout.py

```diff
@@ -1,73 +1,89 @@
 #!/usr/bin/env python
-
 import argparse
 import json
-from models import Workout, Exercise, Session, ExerciseCategory, WorkoutExercise
-from random import random, shuffle, sample
+from models import Workout, Exercise, ExerciseCategory, WorkoutExercise
+import yaml
+from random import random, sample
 import subprocess
-from six import iteritems
 from itertools import groupby, count
+from sqlalchemy.orm import sessionmaker, scoped_session
+from sqlalchemy import create_engine
+import logging
+logging.basicConfig(level=logging.INFO)
+
+class GenerateWorkout(object):
```

Comment:
I'm game to rename the class, but changing the method to `generate` feels confusing vs `generate_exercises_for_category`. Is there some class/method naming convention I should be thinking about?